debian/changelog

```diff
@@ -1,3 +1,27 @@
+ostree (2017.7-1-endless1) unstable; urgency=medium
```

Comment:
⇒ This will become the `Version_2017.7_debian` tag, right? (Not `Version_2017.7-1endless1_debian` — `jenkins-obs` complains about that. Specifically, it seems to unconditionally look for the `Version_2017.7_debian` tag.)